### ISSUE TITLE
Compilation errors on Ubuntu reported by key01023@gmail.com

### ISSUE THREAD

**Issue by svigerske** [coin-or/OBOE#14]
Issue created by migration from Trac.

Original creator: @nsawhney68

Original creation time: 2014-11-28 14:01:58

Assignee: somebody

Version: 

From the email:
So here is what happen, I do:

svn co https://projects.coin-or.org/svn/OBOE/trunk OBOE
cd OBOE
export BLAS=/usr/lib/libblas.a
export LAPACK=/usr/lib/liblapack.a
export LAPACKCPP_DIR=/usr/local/include/lapackpp
export LAPACKCPP_LIB=/usr/local/lib/liblapackpp.la
./configure

up to now everything is great, then i do:

make

but i get the error

In file included from AccpmBlasInterface.h:14:0,
                 from AccpmVector.C:13:
AccpmGenMatrix.h: In member function ‘double& Accpm::AccpmGenMatrix::operator()(int, int) const’:
AccpmGenMatrix.h:86:86: error: invalid initialization of reference of type ‘double&’ from expression of type ‘const double’
     inline double& operator()(int i, int j) const { return RealMatrix::operator()(i,j); }                            

then i don’t know how to edit to make it work but blindly follow one post….. After I edit according to the previous post: inline double& operator()(int i, int j) --->> inline const double& operator()(int i, int j) I get this error:

In file included from AccpmBlasInterface.h:14:0,
                 from AccpmVector.C:13:
AccpmGenMatrix.h:86:26: error: ‘const double& Accpm::AccpmGenMatrix::operator()(int, int)’ cannot be overloaded
     inline const double& operator()(int i, int j) { return RealMatrix::operator()(i,j); }                             
                          ^
AccpmGenMatrix.h:85:26: error: with ‘const double& Accpm::AccpmGenMatrix::operator()(int, int)’
     inline const double& operator()(int i, int j) { return RealMatrix::operator()(i,j); }

so basically the edit does not work for me….

Thank you very much for looking into this… Do you know what happened?

With regards,
Wenxuan

**Comment by svigerske** [comment]
Comment by @nsawhney68 created at 2014-11-28 14:23:06

Changing assignee from somebody to @nsawhney68.

**Comment by svigerske** [comment]
Comment by @nsawhney68 created at 2014-11-28 14:23:06

Changing status from new to assigned.

**Comment by svigerske** [comment]
Comment by @nsawhney68 created at 2014-11-28 14:23:47

The code changes have been submitted in stable 1.0.4.
A release will be created shortly.
